src/coredump/coredump.c

```diff
@@ -1214,6 +1224,74 @@ static int process_kernel(int argc, char* argv[]) {
         return r;
 }
 
+static int process_traceback(int argc, char *argv[]) {
```

Comment:
hmm, i am more used to the word "backtrace" rather than "traceback"? /me is confused...